app/admin/products.rb

```diff
@@ -0,0 +1,119 @@
+include UserUtils
+
+ActiveAdmin.register Product do
+  config.create_another = true
+  actions :all
+  permit_params :id, :user_email, :create_another
```

Comment:
so after doing some digging I discovered that using `permit_params` is problematic because it is intended to only be used with params that pertain directly to the model's attributes. `user_email` was throwing a warning in the console as being disallowed so I opted for 2 traditional strong params methods. It may seem a bit verbose but `active_admin` was meant to be more streamlined and less customized than we are going for, so in order to keep things safe and conventional in regards to params I think this is our best bet.